.ci/sonar.sh

```diff
@@ -0,0 +1,27 @@
+#set checkstyle sonar profile
+curl -X POST -u admin:admin -F 'backup=@config/default_sonar_profile.xml' -v http://localhost:9000/api/profiles/restore
+
+#execute inspection
+mvn sonar:sonar -P sonar -Dsonar.language=java -Dsonar.profile=checksyle-profile
+
+# Uncomment following to get HTML report.
+# mvn sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.issuesReport.html.enable=true \
+#        -Dsonar.language=java -Dsonar.profile=checksyle-profile
+
+
+#get and parse response from sonar
+#give some time to sonar for report processing
+sleep "60"
+curl -u admin:admin -v http://localhost:9000/api/issues/search?componentRoots=com.puppycrawl.tools:checkstyle \
+        > response.json
+
+
+REPORT="$(cat response.json)"
+OUTPUT="$(cat response.json | grep -o -E '\"total\":[0-9]+,' | grep -o -E [0-9]+)"
```

Comment:
please use `jq` shell tool to grep json.
examples are at https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/xtr_pr-description.sh